changelogs/fragments/56629-synchronize-password-auth.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - synchronize - fix password authentication on Python 2 (https://github.com/ansible/ansible/issues/56629)
```

Comment:
Should this changelog mention `run_command()` instead since that is what is actually changing in this PR?

Reply:
Maybe?  Something by extension.

Reply:
I'll add another line about the change to the function.